### ISSUE TITLE
Provide REST Mode

### ISSUE THREAD

**Issue by attila-lin** [kxxt/aspeak#71]
When use websocket in HTTP API, will cause 

```
Connection closed, code: 1000, reason: Exceeded maximun websocket connection idle duration(> 300000ms)
```

**Comment by kxxt** [comment]
Added to milestone v6.0. Though I am not sure about when I will have time to work on it :)

**Comment by kxxt** [comment]
Implemented in [v6.0.0-alpha.2](https://github.com/kxxt/aspeak/releases/tag/v6.0.0-alpha.2). Note that there are some other major breaking changes.

**Comment by attila-lin** [comment]
I tried. It works so nice.
